tests/test_diff.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from vorta.views.diff_result import parse_diff_lines
+
+
+@pytest.mark.parametrize('line, expected', [
+    ('changed link        some/changed/link',
+     (0, 'changed', 'link', 'some/changed')),
+    (' +77.8 kB  -77.8 kB some/changed/file',
+     (77800, 'modified', 'file', 'some/changed')),
+])
+def test_parsing_diff_lines(line, expected):
```

Comment:
This function should probably be in the same file as https://github.com/borgbase/vorta/blob/936d3c10d09cd601351ed49d280a8d8b2aab99cf/tests/test_archives.py#L135

I have no strong feelings regarding which one of the two should be moved